bash_completions.go

```diff
@@ -88,6 +88,11 @@ __handle_reply()
     fi
     COMPREPLY=( $(compgen -W "${completions[*]}" -- "$cur") )
 
+    if [[ ${#completions[@]} -eq 0 ]]; then
+    	__debug "${FUNCNAME}: no known completions, falling back to _filedir"
+    	_filedir
+    fi
+
```

Comment:
I'd rather a commend had to say it would take an arbitrary file as an argument instead of just always doing it. Also, if we do just sorta randomly default to _filedir should it go after the __custom_func handler? I don't want people to start randomly completely filenames and never running the __custom_func...